qiskit/providers/aer/noise/utils/noise_model_inserter.py

```diff
@@ -0,0 +1,45 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Noise model inserter module
+The goal of this module is to add QuantumError gates (Kraus gates) to a circuit
+based on a given noise model. The resulting circuit cannot be ran on a quantum computer
+but can be handled correctly by simulators
+"""
+import qiskit.compiler
+
+
+def add_errors(circuit, noise_model, transpile=False):
```

Comment:
Can you rename this to `insert_noise` and make it so it works with either a single input circuit, or a list of input circuits similarly to`transpile/execute/assemble`

Reply:
Addressed in 72210203ad4bd445cc05a9c505b7328987f0a399